### PR TITLE
[WIP] Postpone "early tests" until the catkin work space is fully setup

### PR DESCRIPTION
**Pull request by rhaschke**
Addresses #80. `catkin_lint` should run only after the workspace has been setup.

### CODE REVIEW

File: travis.sh [hunk 1]
```diff
@@ -60,6 +60,7 @@ function run_docker() {
         -e TRAVIS_BRANCH \
         -e TRAVIS_PULL_REQUEST \
         -e TRAVIS_OS_NAME \
+        -e TEST_PKG \
```
Comment (mlautman) [inline travis.sh]: This seems unrelated. Did we simply miss this param before?

Comment (rhaschke) [reply travis.sh]: As pointed out in https://github.com/ros-planning/moveit_ci/pull/84#issuecomment-519843552, this variable needs to be passed to docker to recognize unit tests there...
So yes, we missed to pass this variable before and yes, it is related to make the unit tests pass again...
